### ISSUE TITLE
Cookies tests added in #1690 started to fail

### ISSUE THREAD

**Issue by Hexcles** [web-platform-tests/wpt.fyi#1751]
```
--- FAIL: TestHandleOauth (0.00s)
    login_test.go:96:
                Error Trace:    login_test.go:96
                Error:          Not equal:
                                expected: 307
                                actual  : 400
                Test:           TestHandleOauth
    login_test.go:97:
                Error Trace:    login_test.go:97
                Error:          Not equal:
                                expected: "/"
                                actual  : ""

                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1 +1 @@
                                -/
                                +
                Test:           TestHandleOauth
    login_test.go:99:
                Error Trace:    login_test.go:99
                Error:          Should be true
                Test:           TestHandleOauth
    login_test.go:100:
                Error Trace:    login_test.go:100
                Error:          Should be true
                Test:           TestHandleOauth
    login_test.go:101:
                Error Trace:    login_test.go:101
                Error:          Should be true
                Test:           TestHandleOauth
    login_test.go:102:
                Error Trace:    login_test.go:102
                Error:          Should be true
                Test:           TestHandleOauth
    login_test.go:103:
                Error Trace:    login_test.go:103
                Error:          Should be true
                Test:           TestHandleOauth
    login_test.go:104:
                Error Trace:    login_test.go:104
                Error:          Should be true
                Test:           TestHandleOauth
    login_test.go:105: missing call(s) to *sharedtest.MockGitHubOAuth.GetNewClient(is anything) /usr/local/google/home/robertma/github/wpt.fyi/webapp/login_test.go:87
    login_test.go:105: missing call(s) to *sharedtest.MockGitHubOAuth.GetGitHubUser(is anything) /usr/local/google/home/robertma/github/wpt.fyi/webapp/login_test.go:88
    login_test.go:105: missing call(s) to *sharedtest.MockGitHubOAuth.GetAccessToken() /usr/local/google/home/robertma/github/wpt.fyi/webapp/login_test.go:90
    login_test.go:105: aborting test due to missing call(s)
```

These tests were added in commit 7f883bc37ba67052d6365a3e2f7cf3b1358c1864 (#1690). And they had been passing until recently.

Example failure on a recent GitHub Action: https://github.com/web-platform-tests/wpt.fyi/pull/1750/checks?check_run_id=399776550#step:3:1962

**Comment by Hexcles** [comment]
I've tried checking out to the earlier revisions that used to pass. And the tests still fail. I suspect there's some kind of timestamp / expiry involved.

**Comment by Hexcles** [comment]
Fixed by #1752
